opm/simulators/wells/WellState.hpp

```diff
@@ -236,6 +257,33 @@ namespace Opm
 
         WellMapType wellMap_;
 
+        using MPIComm = typename Dune::MPIHelper::MPICommunicator;
+#if DUNE_VERSION_NEWER(DUNE_COMMON, 2, 7)
+        using Communication = Dune::Communication<MPIComm>;
+#else
+        using Communication = Dune::CollectiveCommunication<MPIComm>;
+#endif
+        void gatherVectorsOnRoot(std::vector< data::Connection > from_connections,
+                                 std::vector< data::Connection > to_connections,
```

Comment:
Are these supposed to be by value/copies?

Reply:
Thanks, that was indeed a problem.